deployment/postgres-init-db/sql/tables/merlin/external_events/external_event.sql

```diff
@@ -0,0 +1,74 @@
+create table merlin.external_event (
+    key text not null,
+    event_type_name text not null,
+    source_key text not null,
+    derivation_group_name text not null,
+    start_time timestamp with time zone not null,
+    duration interval not null,
```

Comment:
We ultimately want this check to be a part of the external event schema. However, it has been updated with a trigger, so that uploads that fail this check don't go through (see [external_event duration nonnegative check
](https://github.com/NASA-AMMOS/aerie/commit/8f3a324a83f89e0b39527391fcf38021bff06551)).